app/sprites.py

```diff
@@ -55,6 +53,12 @@ def set_visibility(self, visible=True):
     def set_enabled(self, enabled=True):
         self.enabled = enabled
 
+    def __is_mouse_over_button(self, event):
```

Comment:
for private methods / variables, you usually only need to use a single underscore